panel_grid.py

```diff
@@ -46,34 +55,38 @@ def create_panel(self):
         tab = Frame(tablayout)
         tab.pack(fill="both")
 
-        for i in range(6):              # for rows in activities
-            for j in range(8):          # for cols in days
+        for i in range(len(self.activities[self.week-1])+1):              # for rows in activities
```

Comment:
from this line to line 76, needs to move into it's own function which would take "a" week in the timeline. Right now we don't have anything that will cache the Timeline object in the memory, so when we call create_panel() it's gonna be the first week, but once we already have "a" Timeline object in the memory, we could pass it the current week of today, whatever today may be.